force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -378,6 +458,12 @@ public with sharing class ClassesObjectsMethods {
      * @return The Person object of the older person.
      */
     public static Person getOlderPerson(Person person1, Person person2) {
-        return null; // Replace null with the variable you used to store the result
+        if (person1.age > person2.age) {
+            return person1;
+        } else if (person1.age < person2.age) {
+            return person2;
+        } else {
+            return person1;
+        }
```

Comment:
Good work. It can be simplified. If there is a tie you can return either value. So you just need to check if one is greater than the other. 
```
 public static Person getOlderPerson(Person person1, Person person2) {
      /*  return person1.getAge() > person2.getAge() ? person1 : person2;*/
        if (person1.getAge() > person2.getAge()) {
            return person1;
        }
        return person2;

    }
 ```